resources/views/admin/category_management/create_category.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends('admin.layouts.app')
+
+@section('content')
+    <div class="content-wrapper">
+        <div class="table-title">
+            <div class="row">
+                <div class="col-sm-5">
+                    <h2><b>{{ trans('message.add_category') }}</b></h2>
+                </div>
+            </div>
+        </div>
+
+        @if ($errors->any())
+            <div class="alert alert-warning alert-dismissible fade show" role="alert">
+                <ul>
+                    @foreach ($errors->all() as $error)
+                        <li>{{ $error }}</li>
+                    @endforeach
```

Comment:
Nếu có 10 errors thì nó sẽ in ra cả 10 à? 
Có thể handle error theo từng input field đc ko ?

Reply:
tại form này chỉ có 2 input nên e làm cách này, nếu có nhiều input thì e sẽ xử lý handle riêng cho từng input filed ạ

Reply:
resolved